labeled_data/bot/index.yml

```diff
@@ -0,0 +1,12 @@
+name: GitHub Bot
+type: Bot
+label:
+  - github_user
+github_user:
```

Comment:
The format is not correct, this line is redundant, and following lines need a more `tab`